widget/select_entry.go

```diff
@@ -80,10 +80,10 @@ func (e *SelectEntry) Resize(size fyne.Size) {
 // SetOptions sets the options the user might select from.
 func (e *SelectEntry) SetOptions(options []string) {
 	e.options = options
-	var items []*fyne.MenuItem
-	for _, option := range options {
+	items := make([]*fyne.MenuItem, len(options))
+	for i, option := range options {
 		option := option // capture
```

Comment:
With the addition of `i` I think you can remove the capture and use `options[i]` in place of `option`

Reply:
Hmm, not quite. It looks like the captures are still needed when doing it that way as test are failing otherwise.

Reply:
Did you change the second occurrence of `option`? the first one won't matter but the one in `SetText` is what I meant.

Reply:
Yes. I removed the item that we get from the range and replaced both occurrences with options[i]. This caused the tests to fail.

Reply:
Oh yes I see now, the `i` would have to have been captured. Apologies for the noise!

Reply:
No worries.